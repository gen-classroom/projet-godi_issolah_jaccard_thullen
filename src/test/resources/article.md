# Un titre

Le contenu de la premi√®re page de mon site!

*Avec une liste:*

- Un
- deux
- 3

Et un [lien](https://www.youtube.com/watch?v=dQw4w9WgXcQ)