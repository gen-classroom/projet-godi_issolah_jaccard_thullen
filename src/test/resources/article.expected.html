<html lang="en">
<head>
    <meta charset="utf-8">
    <title>Mon site | La première page de mon site !</title>
    </head>
<body>
<ul>
    <li><a href="index.html">home</a></li>
    <li><a href="/content/page.html">page</a></li>
</ul><h1>Un titre</h1>
<p>Le contenu de la première page de mon site!</p>
<p><em>Avec une liste:</em></p>
<ul>
<li>Un</li>
<li>deux</li>
<li>3</li>
</ul>
<p>Et un <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">lien</a></p>

</body>
</html>